### DISCUSSION TITLE
Minecraft 1.18

### DISCUSSION THREAD

**Discussion by Borris-Chuck** [SkriptLang/Skript#4453] [Suggestions] [answered]
please update Skript for Minecraft 1.18
Could not load 'plugins\Skript.jar' in folder 'plugins'

**Comment by NotSoDelayed** [comment, marked as answer]
You can grab the tested Skript jar which should work in 1.18 in skUnity Discord server, #skript-news channel.

**Comment by dsdarewfdsgtfsrhytr** [comment]
HELIKOPTER HELIKOPTER
